.travis.yml

```diff
@@ -3,13 +3,20 @@ dist: trusty
 
 language: java
 jdk: oraclejdk8
+
+cache:
+  directories:
+  - $HOME/.m2
+  - $HOME/.p2
+  - extensions/ui/org.eclipse.smarthome.ui.paper/npm_cache
+
 before_install: echo "MAVEN_OPTS='-Xms1g -Xmx2g -XX:PermSize=512m -XX:MaxPermSize=1g'" > ~/.mavenrc
 
 #$ mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
 #$ mvn test -B
 
 install:
-  - echo 'mvn clean install -B -V -Dspotbugs.skip=true 1> .build.stdout 2> .build.stderr' > .build.sh
+  - echo 'mvn clean install -B -V -Dspotbugs.skip=true' > .build.sh
```

Comment:
This reverts #2608 - has anything changed? I.e. has the limit been dropped in the meantime?

Reply:
Thats why the title says WIP, I wanted to check the correct paths of the cache. Will revert this with the next commit.